intl/msg_hash_us.h

```diff
@@ -731,9 +731,9 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_POLL_TYPE_BEHAVIOR_NORMAL,
 MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_PREFER_FRONT_TOUCH,
       "Prefer Front Touch")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_REMAPPING_DIRECTORY,
-      "Input Remapping Dir")
+      "Input Remapping (dir)")
```

Comment:
I hate Dir so much. I feel that I made the lesser of two evils. I think it's required because some other settings will use that too. For example, see Main Menu --> Help. `You can put it in Cores (dir).`